pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -156,8 +211,8 @@ type MustGatherOptions struct {
 }
 
 // Run creates and runs a must-gather pod.d
-func (o *MustGatherOptions) Run(rsyncCmd *cobra.Command) error {
-	if len(o.Image) == 0 {
+func (o *MustGatherOptions) Run() error {
+	if len(o.Images) == 0 {
```

Comment:
That is one of those checks that should be in `Validate`.

Reply:
Done